boards/pine64-pinephoneA64/README.md

```diff
@@ -2,6 +2,17 @@
 
 ## Device-specific notes
 
+When using prebuilt distribution images it's wise to strip uBoot from the image.
+With the use of prebuilt images on SD-Card it's mandatory to strip uBoot from it, as uBoot will otherwise interfere with Tow-boot and only boot from SD-Card.
+To strip uBoot from a SD-Card installation I used the Gnome Disks utility, but any other should work as well.
```

Comment:
Don't use terms like "I" in documentation, the reader has no clue who you are and neither should they care.

Also in general I would replace these steps with a general "wipe the first 32MB", maybe alongside a `dd` command which does so (since every Linux user has dd).